### ISSUE TITLE
Request - authorities - ASCII?

### ISSUE THREAD

**Issue by dustymc** [ArctosDB/arctos#9141]
Help us understand your request (check below):

- [x] other

**Describe what you're trying to do** 

Understand need, developed guidelines for consistent data.

[Itaboraí Formation](https://arctos.database.museum/info/ctDocumentation.cfm?table=ctlithostratigraphic_formation#itabora__formation) was created. 

* good: It's correct, that's how it was published.
* not so great: Some users won't find it; Arctos has some bitwise indexes and under them ``í`` != ``i``

I think what happened is probably correct, but we should be consistent (have other things been ASCII-ified?) and this possibly requires development changes (eg fuzzy-match queries).

There's also a question of scale and scope- eg does whatever we do to ctsex_cde also apply to geography, and if so should ``Hau Giang`` be ``Hậu Giang`` (maybe??), or ``China`` ``中华人民共和国`` (probably not?)?

HELP! 

(Seems fairly low priority but adding to AWG milestone for eventual discussion - feel free to redirect!)

------------

Please review the [Arctos Code of Conduct](https://arctosdb.org/code-of-conduct/) to ensure respectful communication, and see <https://handbook.arctosdb.org/how_to/How-to-Use-Issues-in-Arctos.html> for more information about submitting and managing Issues.

**Comment by dustymc** [comment]
See also ctculture eg https://arctos.database.museum/info/ctDocumentation.cfm?table=ctculture#h_n and many more

**Comment by dustymc** [comment]
Discussed https://docs.google.com/document/d/1g1YcZ213u8KAQ-GDHUgSEnqOclkrGR_O5ByWIhfoBpc/edit?tab=t.0

Allow "vaguely Latin-ish" authorities.

TODO: Update handbook

**Comment by dustymc** [comment]
https://handbook.arctosdb.org/documentation/authorities.html updated, closing
